pkg/config/config.go

```diff
@@ -54,6 +54,7 @@ var DefaultConfig = Config{
 	ServerFlags:           server.DefaultFlags,
 	Metrics:               metrics.DefaultConfig,
 	Integrations:          DefaultVersionedIntegrations,
+	DisableSupportBundle:  false,
```

Comment:
One thing I dislike and its probably a broader conversation but that we use both enable* and disable* flag names. Makes it harder to grasp which means which.